.travis.yml

```diff
@@ -1,11 +1,14 @@
 language: node_js
 node_js:
-    - "6"
-    - "7"
     - "8"
 script:
     - npm test
     - node dist/cli.js
+    - jsvu --os=linux64 --engines=all
+    - ~/.jsvu/chakra dist/cli.js
+    - # ~/.jsvu/javascriptcore dist/cli.js
```

Comment:
😢

Reply:
Did you see https://github.com/GoogleChromeLabs/jsvu#supported-engines, and specifically https://bugs.webkit.org/show_bug.cgi?id=179945? I’d love to support JSC on Linux, but I can’t seem to find any downloads for it! Can you help?

Reply:
It's available on [wasm-stat.us](wasm-stat.us), which is built from https://github.com/webassembly/waterfall

Reply:
jsvu has a [policy](https://github.com/GoogleChromeLabs/jsvu#security-considerations) of only trusting installers that come straight from the source, i.e. from Apple directly.

Reply:
@jfbastien Even so, it looks like the Linux builder doesn’t include the “JSC” step :(

Reply:
@mathiasbynens that should be super easy to fix on https://github.com/webassembly/waterfall. I'm sure @dschuff is happy to take the PR for it.

Reply:
https://github.com/WebAssembly/waterfall/pull/299, but this does not mean I am okay with changing jsvu’s policy! ;)

Reply:
@jfbastien In fact, if Apple could provide these downloads directly, WebAssembly/waterfall could just use `jsvu` to get them instead of compiling everything from source. (Same for all the other JS engines.) 👍

Reply:
As discussed on your webkit-dev post, the GTK folks maintain the Linux port. Lucas says we're happy to host the binary.

Apple provides an official jsc with MacOS.